docs/authoring.md

```diff
@@ -23,45 +25,68 @@ Accepted `<TargetFramework>` properties |
 `net5.0-windows10.0.17763.0` |
 
 The library you are authoring should specify the following properties in its project file: 
-```
-  <PropertyGroup>
-    <!-- update the Windows API version to reflect your TargetFramework -->
-    <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>
-    <CsWinRTComponent>true</CsWinRTComponent>
-    <CsWinRTEnableLogging>true</CsWinRTEnableLogging>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <GeneratedFilesDir Condition="'$(GeneratedFilesDir)'==''">$([MSBuild]::NormalizeDirectory('$(MSBuildProjectDirectory)', '$(IntermediateOutputPath)', 'Generated Files'))</GeneratedFilesDir>
-  </PropertyGroup>
+``` csproj
+<PropertyGroup>
+  <!-- update the Windows API version to reflect your TargetFramework -->
+  <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>
+  <CsWinRTComponent>true</CsWinRTComponent>
+  <CsWinRTEnableLogging>true</CsWinRTEnableLogging>
+  <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  <GeneratedFilesDir Condition="'$(GeneratedFilesDir)'==''">$([MSBuild]::NormalizeDirectory('$(MSBuildProjectDirectory)', '$(IntermediateOutputPath)', 'Generated Files)</GeneratedFilesDir>
+</PropertyGroup>
 ```
 And don't forget to include a `PackageReference` to `Microsoft.Windows.CsWinRT`!
 
 
-## Known Authoring Issues
-1. There are some programs you could write as/in your component that aren't available in the Windows Runtime. 
-We are working on implementing diagnostics in our tool that will catch these errors before a winmd is generated for your component.
-
-2. Not all C# types have been mapped to WinRT types, but we are working on completing this coverage. 
-
-3. Composable type support is still in progress
+## Using your authored component
+To use the component in a C# app, the authored component just needs to be added as a project/package reference.
 
-## Using the authored component
+For native (C++) apps, there are DLLs needed to host your authored component. When you package your runtime component, they are automatically
+added to your nupkg, before the ```GenerateNuspec``` MSBuild step.  
 
-To use the component in a C# app, the authored component just needs to be added as a project/package reference.
+You will need to create a targets file for your component, if you are not already, that imports a CsWinRT targets file. The imported targets file configures the native app to use the hosting dlls at runtime.
+This means for your component ```MyAuthoredComponent```, you will need to a targets file that has an import statment for ```MyAuthoredComponent.CsWinRT.targets```. 

```

Comment:
will need to a targets -> will need a targets

Reply:
nice catch! thx